src/Germany/Mews.Fiscalizations.Germany/V2/FiskalyClient.cs

```diff
@@ -91,18 +96,18 @@ public Task<ResponseResult<Transaction>> StartTransactionAsync(AccessToken token
         {
             return Client.ProcessRequestAsync<Dto.TransactionRequest, Dto.TransactionResponse, Transaction>(
                 method: HttpMethod.Put,
-                endpoint: $"tss/{tssId}/tx/{transactionId}",
+                endpoint: $"tss/{tssId}/tx/{transactionId}?tx_revision=1",
```

Comment:
The user of the client should be able to control the revision. So I would either:
1. Allow specifying the revision through a param.
2. Include the current revision in the response. Saying that a transaction always starts with revision 1 is a good solution, so I would probably do this.

Reply:
We can do it in a separate PR, but i mentioned this in the Readme and it is also mentioned in Fiskaly documentation.